lib/logger.js

```diff
@@ -47,7 +47,19 @@ const serializers = {
       remotePort: req.connection.remotePort
     }
   },
-
+  err: function asErrValue (err) {
+    var obj = {
+      type: err.constructor.name,
+      message: err.message,
+      stack: err.stack
+    }
+    for (var key in err) {
+      if (obj[key] === undefined) {
+        obj[key] = err[key]
+      }
+    }
+    return obj
+  },
```

Comment:
you can use `pino.stdSerializers.err`

Reply:
I wanted to use it but it creates a hard binding to the module.

Reply:
this same file is already requiring it: https://github.com/StarpTech/fastify/blob/e1bde9c123165b03cd372ceddf97d38d122c59da/lib/logger.js#L9.

Reply:
Fixed